apps/pre-award/copilot/environments/addons/funding-service-magic-links.yml

```diff
@@ -68,12 +68,12 @@ Resources:
         IncludeSpace: false
         PasswordLength: 16
 
-  # Creation of the cluster itself
+  # Creation of the cluster itself (Funding Service Magic Links)
```

Comment:
Can we change the comment as we've changed the name in the code?

Reply:
I have created branches for the service changes as well. I will raise the PR now